src/components/templates/AnimateMap/game/map/systems/ClickableSpriteSystem.ts

```diff
@@ -21,6 +21,7 @@ export class ClickableSpriteSystem extends System {
     this.players = engine.getNodeList(PlayerMovementNode);
 
     this.container.on("pointerdown", this.handlePointerDown, this);
+    this.container.on("pointerdown", this.handlePointerDown, this);
```

Comment:
Why is this line duplicate? Could it be a typo?

Reply:
yes, made that accidentially
fixed in https://github.com/sparkletown/sparkle/pull/2161/commits/470154cca18bb5273e216b7d43acaace4ac25b4a